src/main/java/uk/gov/ons/ctp/response/action/service/ActionProcessingService.java

```diff
@@ -93,20 +93,14 @@ public void processActionRequest(final Action action) throws CTPException {
       final ActionRequest actionRequest = prepareActionRequest(action);
 
       if (actionRequest != null) {
-        if (actionType.getHandler().equals("Field")) {
-          if (actionRequest.getCaseGroupStatus().equals("NOTSTARTED")
-              || actionRequest.getCaseGroupStatus().equals("INPROGRESS")) {
-            actionInstructionPublisher.sendActionInstruction("Field", actionRequest);
-          }
-        } else {
-          actionInstructionPublisher.sendActionInstruction(actionType.getHandler(), actionRequest);
-        }
-        // advise casesvc to create a corresponding caseevent for our action
-        caseSvcClientService.createNewCaseEvent(action, CategoryDTO.CategoryName.ACTION_CREATED);
-      } else {
-        log.with("action_id", action.getId())
-            .error("Unexpected situation. actionType is not defined for action");
+        actionInstructionPublisher.sendActionInstruction(actionType.getHandler(), actionRequest);
       }
+
+      // advise casesvc to create a corresponding caseevent for our action
+      caseSvcClientService.createNewCaseEvent(action, CategoryDTO.CategoryName.ACTION_CREATED);
```

Comment:
Actions are not created at this point so this feels like the wrong place to send this case event. 
The action processing service creates action requests from already existing actions.